src/components/results/dynamicsimulation/types/dynamic-simulation-result.type.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+// This type is derived from the {@link com.powsybl.timeseries.TimeSeriesMetadata}
+export type TimeSeriesMetadata = {
+    name: string;
+};
+
+// This type is taken from the {@link com.powsybl.dynamicsimulation.TimelineEvent}
+export type TimelineEvent = {
+    time: number;
+    modelName: string;
+    message: string;
+};
```

Comment:
This type is not used anywhere else, so I suggest to directly create a type for the key values

Reply:
Event this type is not used at this moment (in future we can use it for typing in fetching service or even in the component which shows the timeline), but always useful since it modelizes the structure of result data from the back..

In my opinion,  we can keep this type.